### PR TITLE
FLUO: convert to Helm chart

### PR DESCRIPTION
**Pull request by invidian**
So all components are consistently sourced from Helm charts.

Signed-off-by: Mateusz Gozdek <mateusz@kinvolk.io>

### CODE REVIEW

File: assets/charts/components/flatcar-linux-update-operator/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+# Application charts are a collection of templates that can be packaged into versioned archives
+# to be deployed.
+#
+# Library charts provide useful utilities or functions for the chart developer. They're included as
+# a dependency of application charts to inject those utilities and functions into the rendering
+# pipeline. Library charts do not define any templates and therefore cannot be deployed.
+type: application
+
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
```
Comment (knrt10) [inline assets/charts/components/flatcar-linux-update-operator/Chart.yaml:16]: Do we need these autogenerated comments?

Comment (invidian) [reply assets/charts/components/flatcar-linux-update-operator/Chart.yaml:16]: Hmm, this is something we should add to #883. I always prefer to leave it, as it is the "default" generated by Helm.

File: assets/charts/components/flatcar-linux-update-operator/Chart.yaml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+# Versions are expected to follow Semantic Versioning (https://semver.org/)
+version: 0.1.0
+
+# This is the version number of the application being deployed. This version number should be
+# incremented each time you make changes to the application. Versions are not expected to
+# follow Semantic Versioning. They should reflect the version the application is using.
+appVersion: 1.16.0
```
Comment (knrt10) [inline assets/charts/components/flatcar-linux-update-operator/Chart.yaml:23]: https://github.com/kinvolk/lokomotive/pull/704#discussion_r469104604. Should we match this with our lokoctl or flatcar version?

Comment (invidian) [reply assets/charts/components/flatcar-linux-update-operator/Chart.yaml:23]: We can, with https://github.com/kinvolk/flatcar-linux-update-operator version.

Comment (invidian) [reply assets/charts/components/flatcar-linux-update-operator/Chart.yaml]: Fixed.

**Review by knrt10 — APPROVED** [review]
+1

**Review by surajssd — APPROVED** [review]
ok

**Pull request merged**